packages/get-starknet/src/snap.ts

```diff
@@ -109,7 +110,7 @@ export class MetaMaskSnap {
     senderAddress: string,
     txnInvocation: AllowArray<Call>,
     abis?: Abi[],
-    invocationsDetails?: InvocationsDetails,
+    invocationsDetails?: UniversalDetails,
   ): Promise<InvokeFunctionResponse> {
```

Comment:
it looks like it is not sync with the snap

Reply:
we keep the snap in sync with invocationsDetails for now and fix in the other PR